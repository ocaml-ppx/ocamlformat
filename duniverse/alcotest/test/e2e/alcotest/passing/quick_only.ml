let () =
  let open Alcotest in
  let id () = () in
  run ~argv:[| ""; "--quick" |] __FILE__
    [
      ("test-a", [ test_case "Quick" `Quick id; test_case "Slow" `Slow id ]);
      ("test-b", [ test_case "Slow" `Slow id; test_case "Quick" `Quick id ]);
    ]
