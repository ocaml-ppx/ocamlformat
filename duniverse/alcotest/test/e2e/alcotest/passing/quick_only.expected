Testing `test/e2e/alcotest/passing/quick_only.ml'.
This run has ID `<uuid>'.

  [OK]          test-a          0   Quick.
  [SKIP]        test-a          1   Slow.
  [SKIP]        test-b          0   Slow.
  [OK]          test-b          1   Quick.

Full test results in `<build-context>/_build/_tests/<test-dir>'.
Test Successful in <test-duration>s. 2 tests run.
